FVCOM.md

```diff
@@ -0,0 +1,231 @@
+
+
+    wget http://fvcom.smast.umassd.edu/releases/fvcom-4.3.tar.gz
+2. Decompress the WRF installation package:
+
+
+    tar -zxvf fvcom-4.3.tar.gz
+3. Run the following command to switch to WRF source code directory:
+
+    cd fvcom-4.3
+
+4. Run the following commands to configure related files:
+
+    cp Examples/Estuary/make.inc_example FVCOM_source/make.inc
+    ln -sf FVCOM_source/make.inc ./
+
+5. Edit the make.inc file:
+
+    vim make.inc
+
+   Line 51:
+    TOPDIR = /path/to/FVCOM/FVCOM4.1/FVCOM_source
+   Line 79,80:
+    LIBDIR = -L$(INSTALLDIR)/lib -L../METIS_source/metis -L./libs/julian
+    INCDIR = -I$(INSTALLDIR)/include -I../METIS_source/metis -I./libs/julian
+   Line 97:
+    IOLIBS = -L/path/to/NETCDF/lib -L/path/to/HDF5/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5 -lz -lcurl -lm
+   Line 989:
+    IOINCS = -I/path/to/NETCDF/include -I/path/to/HDF5/include
+   Line 532:
+    #  Intel/MPI Compiler Definitions (SMAST)
+    #--------------------------------------------------------------------------
+    #         CPP      = /usr/bin/cpp
+    #         COMPILER = -DIFORT
+    #         CC       = mpicc
+    #         CXX      = mpicxx
+    #         CFLAGS   = -O3
+    #         FC       = mpif90
+    #         DEBFLGS  = -check all -traceback
+    # Use 'OPT = -O0 -g'  for fast compile to test the make
+    # Use 'OPT = -xP' for fast run on em64t (Hydra and Guppy)
+    # Use 'OPT = -xN' for fast run on ia32 (Salmon and Minke)
+    #         OPT      = -O0 -g
+    #         OPT      = -axN -xN
+    #         OPT      = -O3
+    
+    # Do not set static for use with visit!
+    #         VISOPT   = -Wl,--export-dynamic
+    #         LDFLAGS  =  $(VISITLIBPATH)
+    #--------------------------------------------------------------------------
+    #  gfortran defs
+    #--------------------------------------------------------------------------
+             CPP      = /usr/bin/cpp
+             COMPILER = -DGFORTRAN
+             CC       = mpicc
+             CXX      = mpicxx
+             FC       = mpif90
+             DEBFLGS  =
+             OPT      = -O3 -ffixed-line-length-none -ffree-form -ffree-line-length-none
+             CLIB     =
+
+
+6. Run the following command to compile the METIS:
+
+    cd ./METIS_source
+    tar -zxvf metis.tgz
+    wget https://www.math-linux.com/IMG/patch/metis-4.0.patch --no-check-certificate
+    cd metis
+    patch -p2 < ../metis-4.0.patch
+    mkdir -p /path/to/FVCOM/FVCOM4.3/FVCOM_source/libs/install/lib
+    mkdir -p /path/to/FVCOM/FVCOM4.3/FVCOM_source/libs/install/include
+    mkdir -p /path/to/FVCOM/FVCOM4.3/FVCOM_source/libs/install/bin
+    make -j
+    make install
+
+7. Run the following command to compile the julian:
+
+    cd ../../FVCOM_source/libs
+    tar -zxvf julian.tgz
+    cd julian
+    make -j
+    make install
+
+8. Run the following command to modify some source code:
+
+    cd /path/to/FVCOM/FVCOM4.1/FVCOM_source
+
+    vim mod_newinp.F
+   Line 343:
```

Comment:
Would it be better if a .patch file is created and put in this Git repo next to FVCOM.md file ?